shared/cfg/guest-os/Linux/Fedora/25.x86_64.cfg

```diff
@@ -0,0 +1,27 @@
+- 25.x86_64:
+    image_name = images/f25-64
+    vm_arch_name = x86_64
+    os_variant = fedora25
+    # Installation works fine with mem=1024 on methods such as cdrom
+    # but fails ("No space left on device") with methods such as url.
+    # anyway, let's use a safe default
+    mem = 2048
```

Comment:
I know some tests are overriding the `mem` value and this would override their setting. How about changing the default in `base.cfg` instead? (I'd like to avoid problems where the test declares 32G but this changes it to 2G)